tests/integration/security/authorization_test.go

```diff
@@ -618,16 +616,17 @@ func TestAuthorization_EgressGateway(t *testing.T) {
 	framework.NewTest(t).
 		Features("security.authorization.egress-gateway").
 		Run(func(t framework.TestContext) {
-			ns := namespace.NewOrFail(t, t, namespace.Config{
-				Prefix: "v1beta1-egress-gateway",
-				Inject: true,
-			})
-
-			var a, b, c echo.Instance
+			ns := apps.Namespace1
+			rootns := newRootNS(t)
+			a := apps.A.Match(echo.Namespace(apps.Namespace1.Name()))
+			b := apps.B.Match(echo.Namespace(apps.Namespace1.Name()))
+			vm := apps.VM.Match(echo.Namespace(apps.Namespace1.Name()))
+			// Set up workload X that is accessible via egress gateway.
+			var x echo.Instance
+			XSvc := "x"
 			echoboot.NewBuilder(t).
```

Comment:
It seems you are re-using `a, b` that already existed. Can we create `x` at the suite-level? Tests will be a bit faster if `x`'s creation happens in the same builder as everything else. 

Also do we absolutely need `x`? It seems like a mostly plain pod, so I'd imagine there's something at the suite level we can re-use here.

Reply:
We don't need extra workload as external service. I have reused workload c as destination and removed x.